pkg/grid/src/state/api.ts

```diff
@@ -9,12 +9,13 @@ declare global {
 const api =
   import.meta.env.MODE === 'mock'
     ? {
-        poke: () => {},
-        subscribe: () => {},
-        subscribeOnce: () => {},
-        ship: ''
-      }
+        poke: async () => {},
+        subscribe: async () => {},
+        subscribeOnce: async () => {},
+        ship: '',
+        scry: async () => {}
+      } as unknown as Urbit
     : new Urbit('', '');
 
-api.ship = window.ship;
+api.ship = 'marzod';
```

Comment:
this might have been left on accident

Reply:
Yeah I'm struggling to get vite to import the `session`.js` from the server, any ideas are welcome